pkg/inframanager/p4/cni_es2k.go

```diff
@@ -19,225 +19,170 @@ package p4
 import (
 	"context"
 	"fmt"
+	"net"
+
 	"github.com/antoninbas/p4runtime-go-client/pkg/client"
 	"github.com/ipdk-io/k8s-infra-offload/pkg/inframanager/store"
+	"github.com/ipdk-io/k8s-infra-offload/pkg/types"
 	log "github.com/sirupsen/logrus"
-	"net"
 )
 
-var Env string
-var P4w P4RtCWrapper
+var (
+	cni_table_names = []string{"k8s_dp_control.arp_to_port_table",
+		"k8s_dp_control.pod_gateway_mac_mod_table",
+		"k8s_dp_control.ipv4_to_port_table_tx",
+		"k8s_dp_control.ipv4_to_port_table_rx",
+		"k8s_dp_control.ipv4_to_port_table_tx_tcp",
+		"k8s_dp_control.ipv4_to_port_table_tx_service"}
+
+	cni_action_names = []string{"k8s_dp_control.set_dest_vport",
+		"k8s_dp_control.update_src_dst_mac",
+		"k8s_dp_control.set_dest_mac_vport",
+		"k8s_dp_control.set_dest_vport",
+		"k8s_dp_control.set_dest_mac_vport",
+		"k8s_dp_control.set_dest_mac_vport"}
+)
```

Comment:
Yes agree, this can be improved. we can completely remove these slices and depend on cniupdatemap or svcupdatemap.
Will take it as a future enhancement.